api/resources_portal/models/associations/organization_user_association.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+
+from resources_portal.models.organization import Organization
+from resources_portal.models.user import User
+
+
+class OrganizationUserAssociation(models.Model):
+
+    organization = models.ForeignKey(
+        Organization, blank=False, null=False, on_delete=models.CASCADE
+    )
+    user = models.ForeignKey(User, blank=False, null=False, on_delete=models.CASCADE)
+
```

Comment:
do we want to add timestamps for associations so we can get metrics for when a user joined an organization?

Reply:
That's a good idea, I'll add a created_at but no updated_at since there's nothing to be updated.